app/routes/synonyms/synonyms.server.ts

```diff
@@ -347,14 +350,19 @@ export async function groupMembersByEventId({
 
 async function getSynonymMembers(eventId: string) {
   const db = await tables()
-  const { Items } = await db.circulars.query({
-    IndexName: 'circularsByEventId',
-    KeyConditionExpression: 'eventId = :eventId',
-    ExpressionAttributeValues: {
-      ':eventId': eventId,
-    },
-  })
-  return Items as Circular[]
+  const Items: Circular[] = (
+    await db.circulars.query({
+      IndexName: 'circularsByEventId',
+      KeyConditionExpression: 'eventId = :eventId',
+      ExpressionAttributeValues: {
+        ':eventId': eventId,
+      },
+    })
+  ).Items
+
+  Items.sort((a, b) => b.createdOn - a.createdOn)
+
+  return Items
```

Comment:
Could we just set the sort key on this index?

Reply:
On the circulars table?

Reply:
On this index for the circulars table.

Also, I don't think that this addresses #2877. The problem that issue describes is that the event names are in arbitrary order.

I think that this PR puts the circular IDs in chronological order _within an event_, which is also important, but does not address the issue.

Reply:
It does both, assuming we want to show the group with the most recent circular at the top

Reply:
We want the circulars in descending order within a group as this does.  But I'd like to also see the groups/events also in reverse order of date.  I thought the way to do that was to sort the groups in reverse order of the first circular in that group, which should be a good proxy for date of the event.

Reply:
> It does both, assuming we want to show the group with the most recent circular at the top

How does it sort the event names chronologically?